spec/factories/authors.rb

```diff
@@ -0,0 +1,13 @@
+require 'faker'
+
+FactoryGirl.define do
+  factory :author do
+    name { Faker::Name.first_name }
+    surname { Faker::Name.last_name }
+  end
+
+  factory :no_attr_author, class: Author do
+    name nil
```

Comment:
Do we need Nil factories? Book.new makes the same thing.
BTW. You can nest Factories to make class implicit i.e.

```ruby
FactoryGirl.define do
  factory :author do
    name  'John'
  
    factory :steward do
      last_name 'Steward'
    end
  end
end
```
So creating object form factory :steward you will get Author 'John Steward'